### ISSUE TITLE
[a11y]: Reporting Period Ending Drop Down can not be navigated using the keyboard

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3756]
### Why is this issue important?

Keyboard and Screen Reader users would not be able to select a different reporting period

### Current State

When user visits [ARPA Reporting home page](http://localhost:8080/arpa_reporter/) - They are not able to navigate the drop down using keyboard.

### Expected State

User should be able to navigate and select one of the items in the drop down list using the keyboard only.

### Implementation Plan

Review how the drop down is coded and update so it can be navigated using the keyboard only.

![image](https://github.com/user-attachments/assets/c022f0e8-79ae-46c6-931c-895bf61c6556)

[Bootstrap docs for dropdown](https://getbootstrap.com/docs/4.0/components/dropdowns/
)

**Comment by ClaireValdivia** [comment]
just tested and this looks great - i'm able to select from the dropdown using keyboard commands now. thanks @lsr-explore !
